### DISCUSSION TITLE
BunAdmin - Strapi Demo - Uploader + ListSelector + Notice + MultipleSelector

### DISCUSSION THREAD

**Discussion by Chris533** [strapi/strapi#8007] [Legacy Topics]
Hi, the following uses a small amount of code.

Based on `bunadmin`: Strapi auth plugin,Strapi upload plugin and Strapi data source.
Use components: ListSelector, Uploader, Notice, MultipleSelector.

![preview](https://gblobscdn.gitbook.com/assets%2F-M1ZbjnBaWO_NJOdj8_A%2F-MHlKrSo5A7uYDJDV45k%2F-MHlKxF4-lohTzN3gsiA%2Fblog-post-strapi.png)

## Online demo

[http://blog.eg.bunadmin.com/](http://blog.eg.bunadmin.com/blog/post)

Login with any user below (different menus)

- Username: `admin`, `reviewer`, `user`
- Password: `bunadmin`

## [Getting-started/remote-data](http://blog.eg.bunadmin.com/docs/getting-started/remote-data)

If you have any suggestions, you can reply here.

If you like it after the test please give me a star, thanks.

### [BunAdmin](https://github.com/bunred/bunadmin)

**Comment by derrickmehaffy** [comment]
This looks awesome @Chris533 what frontend tech is it using React, Vue, Angular?

**Reply by Chris533** [reply]
> This looks awesome @Chris533 what frontend tech is it using React, Vue, Angular?

Hi, DMehaffy, thanks, it using `React` + `Material UI`, I wrote three bunadmin plugins for strapi, so the amount of demo blog code is small.

[bunadmin-auth-strapi](https://github.com/bunred/bunadmin/tree/master/plugins/bunadmin-auth-strapi)
[bunadmin-upload-strapi](https://github.com/bunred/bunadmin/tree/master/plugins/bunadmin-upload-strapi)
[bunadmin-source-strapi](https://github.com/bunred/bunadmin/tree/master/packages/bunadmin-source-strapi)

**Reply by derrickmehaffy** [reply]
The filtering you've implemented in the data tables, that is server side (aka strapi side) yeah? 

Your initial demo looks awesome, though I noticed it doesn't look like you've currently implemented a pop-up editor for larger entries (like blog posts).

**Reply by Chris533** [reply]
Yes, it is the real data called from the strapi backend.

Fortunately, you remind me of the content editor, I will improve the demo editor with [Drawer](http://blog.eg.bunadmin.com/docs/components/drawer).

**Reply by derrickmehaffy** [reply]
But seriously, very well done this is starting to look a lot better than some alternatives I've seen like https://github.com/marmelab/react-admin

**Reply by Chris533** [reply]
Thanks, I will continue to improve.

**Reply by Chris533** [reply]
@derrickmehaffy I added `rich text editor` and `bulk update` to the demo just now. [LINK](http://blog.eg.bunadmin.com/blog/post)

**Reply by derrickmehaffy** [reply]
Nice job :+1:

**Reply by Nemzytch** [reply]
Have to say too that I really like your job, will you just keep us posted here ? or do you have any social or anything ? Your stuff looks really nice tbh.

**Comment by Chris533** [comment]
@Nemzytch Glad you like it :),  my social is github lol, we can post here allways.

**Reply by derrickmehaffy** [reply]
@Chris533 just a heads up we plan to decommission the GitHub discussions sometime in 2021, if you want though we could move this over to the https://forum.strapi.io/ maybe in https://forum.strapi.io/c/show-off ?
